### PR TITLE
Adds embed_chance for d4

### PR DESCRIPTION
**Pull request by Bawhoppen**
They already hurt you if you step on them, let's go all the way

### CODE REVIEW

**Comment by Jordie0608** [comment]
Not realistic, thrown d4s aren't sharp enough to embed.

Source: Party wizard magic missile'd the DM

**Comment by Bawhoppen** [comment]
Canon explanation: After years of atrophy from life in deepspace, human skin as become much more less durable and more fragile, to where even a dull object can puncture their flesh.

**Comment by Mercenaryblue** [comment]
A d4 sharp enough to embed in your foot would never be sold on the markets.

**Comment by Arianya** [comment]
image

**Comment by nicbn** [comment]
Let it be 10 at least.

**Comment by kilkun** [comment]
but spacemen are pretty much miniprimarchs with how durable they are.

they can fight naked without air in space for minutes, and around double that if they have an air supply. they easily survive point blank headshots from 12 gauge shotguns, magnums, antimateriel rifles, etc.

You ever been hit with a toolbox in the head? yeah a single blow'd probably do you in, or at the very least give you a concussion. A toolbox to the head just pisses off a spaceman. And these are toolboxes being weilded by other spacemen. Do you know how strong a spaceman is? Without exerting any effort, they can haul around literal tons of iron and other minerals, almost enough to build a new station. They easily drag and throw each other's abhorrently muscled bodies around as easily as they throw paper and pens. For fucks sakes, they can even break these cuffs. 

And you want to say these superhuman assholes have brittle skin? My lad these D4 have to be monomolecular plasteel. There is no other explanation for this shit.

**Comment by Ratvar** [comment]
These D4 have are monomolecular plasteel.

**Comment by The-clown-known-as-Pepe** [comment]
just rename d4 to caltrops tbh

image

**Comment by optimumtact** [comment]
I don't see the point of this, you can step on a die and it hurts you, that doesn't mean it will stick in your flesh

**Comment by Bawhoppen** [comment]
So will anyone want to merge it or not?

**Comment by Bawhoppen** [comment]
The main reason I want it, is that it's another one of those small details that make the game more interesting. Imagine some kind of interesting scenario (as rare as it would be) where you'd have to actually use these in combat.

**Comment by nicbn** [comment]
No

File: code/game/objects/items/dice.dm [hunk 1]
```diff
@@ -64,6 +64,9 @@
 	desc = "A die with four sides. The nerd's caltrop."
 	icon_state = "d4"
 	sides = 4
+	force = 3
+	embed_chance = 40
```
Comment (nicbn) [inline code/game/objects/items/dice.dm]: 40 is too much

**Review by nicbn — CHANGES_REQUESTED** [review]

File: code/game/objects/items/dice.dm (hunk 1 above)
Comment (Bawhoppen) [reply code/game/objects/items/dice.dm]: That's just the chance to actually embed though, nothing to do with damage

Comment (nicbn) [reply code/game/objects/items/dice.dm]: Yeah I know

**Comment by Bawhoppen** [comment]
Would people prefer if I made it actually embed in your foot when you step on it instead?

**Comment by jughu** [comment]
Sure

**Comment by jughu** [comment]
AUCTALLY do that to glass shards since they dont have it and it kinda makes sense

**Comment by KorPhaeron** [comment]
Nobody seems interested in merging this

**Comment by Bawhoppen** [comment]
did you not read my comment about how I was going to change what this PR was for?

**Comment by PKPenguin321** [comment]
Just make a new one

**Comment by Bawhoppen** [comment]
That's probably what I'll do considering I'm on everyone's ignore list apparently

**Comment by optimumtact** [comment]
nobody seems interested in your new one either

**Pull request closed without merging**